### DISCUSSION TITLE
looking to add ethernet conectivity to board ( are there any free Digital pins for chip select )?

### DISCUSSION THREAD

**Discussion by opencodesdev** [sanni/cartreader#987] [Hardware]
are there any free Digital pins for chip select

**Comment by Ancyker** [comment]
There are not, at least not on this current revision anyway. If you wait for the new revision which allows for placing the ATmega2560 directly onto the main PCB (and do that), then there are 6 digital IO pins available. These are recovered pins and cannot be referenced via the Arduino lib. They are pins PJ2, PJ3, PH2, PH7, PG3, and PG4.

Note that adding anything useful via Ethernet might require disabling some cores/features as the mega is already stretched pretty thin on PROGMEM usage.
